packages/orbit-components/src/primitives/TooltipPrimitive/components/TooltipContent.tsx

```diff
@@ -105,54 +105,57 @@ const TooltipContent = ({
     [onClose, onCloseMobile, elements.floating],
   );
 
+  const handleCombinedClick = ev => {
+    handleInnerClick(ev);
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+    onClick(ev);
+  };
+
   return (
-    // Disabling because the onClick exists just to stop propagation of events
+    // Disabling because the onClick exists to close tooltip when clicking in interactibe elements, which should not happen with keyboard
     // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions
-    <div className="w-full" role="tooltip" id={tooltipId} data-test={dataTest} onClick={onClick}>
-      {/* Disabling because the onClick exists to close tooltip when clicking in interactibe elements, which should not happen with keyboard */}
-      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions */}
+    <div
```

Comment:
In this file, I removed `<div>` wrapper and the  `onClick` event on child `div` calls `handleCombinedClick`. This function includes two functions

- `onClick` - passed from the parent
- `handleInnerClick`